### PR TITLE
added scripts for kie, uberfire and dashbuilder release

### PR DESCRIPTION
**Pull request by mbiarnes**

### CODE REVIEW

File: script/release/dash_createAndDeploy.sh [hunk 1]
```diff
@@ -0,0 +1,52 @@
+# removing dashbuilder artifacts from local maven repo (basically all possible SNAPSHOTs)
```
Comment (psiroky) [inline script/release/dash_createAndDeploy.sh]: Do we want to have these dash and uf scripts here? Shouldn't they be in the dashbuilder and uberfire repos?

**Comment by psiroky** [comment]
I would remove all the scripts which are just downloading the -project-sources.tar.gz. We should either have one single generic script which accepts parameters or move this directly to the repo with the DSL scripts. @mbiarnes WDYT?

**Comment by mbiarnes** [comment]
@psiroky can we merge this and do slight changes in future?

**Comment by psiroky** [comment]
@mbiarnes I am not really happy about this, but I guess we don't have any other option at this point. So go ahead and merge this.

**Comment by mbiarnes** [comment]
@psiroky I promise that you will be happy one day ;)

**Pull request merged**